karma.conf.js

```diff
@@ -22,15 +22,7 @@ module.exports = config => {
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: ["jasmine"],
-    plugins: [
-      "karma-jasmine",
-      "karma-coverage",
-      "karma-chrome-launcher",
-      "karma-jasmine-matchers",
-      "karma-spec-reporter",
-      "karma-rollup-preprocessor",
-      "karma-allure-reporter"
-    ],
```

Comment:
I removed these because if plugins is not configured, Karma will automatically detect any installed `karma-*` dependencies.